staging/src/k8s.io/apimachinery/pkg/labels/selector.go

```diff
@@ -74,9 +74,11 @@ type Selector interface {
 	RequiresExactMatch(label string) (value string, found bool)
 }
 
+var everythingSelector Selector = internalSelector{}
```

Comment:
```suggestion
// Sharing this saves 1 alloc per use; this is safe because it's immutable.
var sharedEverythingSelector Selector = internalSelector{}
```

And make the corresponding change to the other variable also, please ("sharedNothingSelector").